amfora.go

```diff
@@ -68,10 +70,34 @@ func main() {
 	display.NewTab()
 	if len(os.Args[1:]) > 0 {
 		display.URL(os.Args[1])
+	} else if !isStdinEmpty() {
+		renderFromStdin()
 	}
 
 	// Start
 	if err = display.App.Run(); err != nil {
 		panic(err)
 	}
 }
+
+func isStdinEmpty() bool {
+	stat, _ := os.Stdin.Stat()
+	return (stat.Mode() & os.ModeCharDevice) != 0
```

Comment:
Is this check portable across OSes? Where did you find this?

Reply:
This check leverages [the Go standard library](https://golang.org/pkg/io/fs/#ModeCharDevice) so I'd have thought it would be. I haven't tested this outside of my Fedora Linux installation so I wouldn't be able to say for sure though.

I got the idea from [here](https://stackoverflow.com/a/26567513/2115499). [This article](https://flaviocopes.com/go-shell-pipes/) explains the approach in a bit more detail.

Reply:
Seems good. [This answer](https://stackoverflow.com/a/41999124/) mentions the same solution working on macOS too. And I guess pipes don't really apply to Windows.